qa/tasks/cephfs/test_volumes.py

```diff
@@ -607,6 +613,43 @@ def test_volume_rename_for_more_than_one_data_pool(self):
         # data pool names unchanged
         self.assertCountEqual(orig_data_pool_names, list(self.fs.data_pools.values()))
 
+    def test_volume_info(self):
+        """
+        Tests the 'fs volume info' command
+        """        
+        subvol_md = ["volume_name", "gid", "size", "subvolumegroups", "pending_subvolume_deletions", "mon_addrs"]
```

Comment:
I think the `test_` should just check if the fields are present for `fs volume info`?

Reply:
This is checked at line: 632

Reply:
I will keep it till line:636, that's fine, I guess?which will be just checking if values initial values are updated as it should be.